publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceServiceUtils.java

```diff
@@ -48,25 +48,29 @@ static Organization userOrganization(UserInstance user) {
         return newOrganization(user.getOrganizationUri());
     }
 
+    public static UserInstance extractOwner(Publication publication) {
+        String owner = publication.getOwner();
```

Comment:
This is the same code as I commented in the handler

Reply:
this is from a Publication. The one in the handler is from RequestInfo.